src/Api/EnumInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Api;
+
+interface EnumInterface
```

Comment:
What do you think about using https://github.com/myclabs/php-enum instead? We already use it in the schema generator.

Reply:
We don't use it yet but we would still need to use this interface unless we have an idea to generate the name for enum without calling `getName()` method.